scripts/kill_switch.ts

```diff
@@ -0,0 +1,68 @@
+// This script calls toggleKillSwitch in the module contract implementing EmailRecoveryManager on zksync sepolia
+// Usage: npx ts-node scripts/kill_switch.ts --private-key <sender-private-key>
+
+import { Command } from 'commander';
+import { Contract, ethers } from 'ethers';
+
+const ZKSYNC_SEPOLIA_RPC_URL = 'https://sepolia.era.zksync.dev';
+const ZKSYNC_SEPOLIA_CHAIN_ID = 300;
+
+// TODO: Please modify the following values according to your specific use case
+const MODULE_ADDRESS = "0x037d80f98aE461DC307F16524EC0Fc53b2E9E0b1"; // For zksync sepolia
```

Comment:
@wshino Could you pass MODULE_ADDRESS as an argument to the script, rather than a fixed value?

Reply:
@SoraSuegami Updated